concepticondata/conceptlists/Tryon-1983-324.tsv

```diff
@@ -249,10 +249,10 @@ Tryon-1983-324-247	247	correct	Adjectives	1725	CORRECT (RIGHT)
 Tryon-1983-324-248	248	rotten	Adjectives	1728	ROTTEN
 Tryon-1983-324-249	249	sick	Adjectives	1847	SICK
 Tryon-1983-324-250	250	smooth	Adjectives	1234	SMOOTH
-Tryon-1983-324-251	251	sore (adj.)	Adjectives	51	SORE
+Tryon-1983-324-251	251	sore (adj.)	Adjectives	2561	HURT (SENSE PAIN)
```

Comment:
I know "feeling sore" is very similar to "feeling hurt" but usually the feeling of being sore is not that severe and you can, for example, move around as usual. Therefore, I'd leave it unmapped. Maybe we'll have more lists with "sore" in the future and can add a distinct concept.

Reply:
Yes, probably better.